script.module.trakt/addon.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <addon id="script.module.trakt"
        name="trakt.py"
-       version="2.5.2"
+       version="2.6.0"
        provider-name="Fuzeman, Razzeee">
   <requires>
     <import addon="xbmc.python" version="2.1.0"/>
```

Comment:
When I did this I think the rule was not really in place and I was much dumber, so this was mostly c&p back then.
Will patch this asap.